scipy/stats/tests/test_distributions.py

```diff
@@ -3520,6 +3520,10 @@ def test_bad_input(self):
         pk = [0.3, 0.3, 0.3, 0.3, -0.2]
         assert_raises(ValueError, stats.rv_discrete, **dict(values=(xk, pk)))
 
+        xk = [1, 1]
+        pk = [0.5, 0.5]
+        assert_raises(ValueError, stats.rv_discrete, **dict(values=(xk, pk)))
```

Comment:
Typically these days we would do, e.g.:
```suggestion
        message = "xk may not contain duplicate values."
        with assert_raises(ValueError, match=message):
            stats.rv_discrete(**dict(values=(xk, pk)))
```
because there are a lot of different `ValueError`s out there and we're interested in one in particular. But this is consistent with the rest of the tests, so it is not worth another CI run.

Reply:
Yes I also prefer using `match`. 

I'm often not sure how much to follow existing patterns vs what I think is best. When working on large codebases that I don't own, I tend towards following patterns, but idk.

Reply:
Of course. Just thought I'd mention for the future.